lib/manager/components/deployment/PeliasPanel.js

```diff
@@ -23,43 +23,30 @@ type Props = {
   project: Project,
   setErrorMessage: typeof setErrorMessage,
   updateDeployment: typeof updateDeployment,
+  updatePelias: typeof updatePelias,
   uploadPeliasWebhookCsvFile: typeof uploadPeliasWebhookCsvFile
 }
 
 type State = {
   fileToDeleteOnSuccesfulUpload: string | null,
-  peliasWebhookUrl: string
 }
 
 class PeliasPanel extends Component<Props, State> {
   state = {
-    fileToDeleteOnSuccesfulUpload: null,
-    peliasWebhookUrl: ''
+    fileToDeleteOnSuccesfulUpload: null
   };
 
```

Comment:
Can you add a comment for_onClickPeliasUpdate?

Reply:
Added in f6d26732